### PR TITLE
docs: use v1.12.0-rc1 images

### PR DESCRIPTION
**Pull request by raphink**
Signed-off-by: Raphaël Pinson <raphael@isovalent.com>

### CODE REVIEW

**Review by michi-covalent — APPROVED** [review]

File: INSTALLATION.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ Containers:       cilium             Running: 2
...
 Cluster Pods:     3/3 managed by Cilium
-Image versions    cilium-operator    quay.io/cilium/operator-aws-service-mesh:v1.11.0-beta.1: 1
-                  hubble-relay       quay.io/cilium/hubble-relay-service-mesh:v1.11.0-beta.1: 1
-                  cilium             quay.io/cilium/cilium-service-mesh:v1.11.0-beta.1: 2
+Image versions    cilium             quay.io/cilium/cilium:v1.12.0-rc1: 2
+                  cilium-operator    quay.io/cilium/operator-generic:v1.12.0-rc1: 1
+                  hubble-relay       quay.io/cilium/hubble-relay:v1.12.0-rc1: 1
 ```
 
 Confirm that the image versions for cilium operator, hubble-relay, and cilium are the [latest beta builds](https://github.com/cilium/cilium-service-mesh-beta#image-tags).
```
Comment (lizrice) [inline INSTALLATION.md:39]: Is this link right? I think we're building from the main repo now so not sure the image tags will be here?

**Review by lizrice — APPROVED** [review]
One question but otherwise great! When this is merged we should notify folks in the Slack channel too and ask them to give this build a try

File: INSTALLATION.md (hunk 1 above)
Comment (raphink) [reply INSTALLATION.md:39]: No, this PR also updates the image versions in the table.

Comment (michi-covalent) [reply INSTALLATION.md:39]: oh yeah you are right. let's ship it.

**Pull request merged**